hack/install.sh

```diff
@@ -372,7 +372,7 @@ do_install() {
 
 			# aufs is preferred over devicemapper; try to ensure the driver is available.
 			if ! grep -q aufs /proc/filesystems && ! $sh_c 'modprobe aufs'; then
-				if uname -r | grep -q -- '-generic' && dpkg -l 'linux-image-*-generic' | grep -q '^ii' 2>/dev/null; then
+				if uname -r | grep -q -- '-generic' && dpkg -l 'linux-image-*-generic' | egrep '^ii|^hi' 2>/dev/null; then
```

Comment:
I'm +1, but I think this should be `grep -qE` (so it stays quiet and so we get expanded correctly):

```
       In addition, the variant programs egrep  and  fgrep  are  the  same  as
       grep -E  and grep -F, respectively.  These variants are deprecated, but
       are provided for backward compatibility.
```